iroh/src/magicsock/transports.rs

```diff
@@ -494,6 +494,7 @@ impl TransportsSender {
         src: Option<IpAddr>,
         transmit: &Transmit<'_>,
     ) -> Poll<io::Result<()>> {
+        let mut found_transport = false;
```

Comment:
If I were to nitpick I'd make this `let mut fallback_ret = Poll::Ready(Ok(()));` and let each transport set this to pending when they hit a pending. But there are many ways to do this and it's subjective and maybe even a nicer way exists.